doc/going-further/extending-castor/events.md

```diff
@@ -6,22 +6,50 @@ application lifecycle.
 
 ## Registering a listener
 
-You can register a listener inside your Castor project by using the
-`Castor\Attribute\AsListener` attribute. This attribute allows you to specify
-the targeted event and the priority of this listener.
+You can register a listener inside your Castor project by using the `Castor\Attribute\AsListener` attribute as a **function** (not a class method). This attribute allows you to specify the targeted event and the priority of this listener.
 
+**Important:** Listener functions must be defined in a file that is imported in your `castor.php` file using the `import()` function.
```

Comment:
Did you face this issue too?

It kinda obvious to me, but I'm biased. I'm a bit reluctant to accept this patch, because if we follow this logic, we have to add this on every page of documentation.